thirdparty/README.md

```diff
@@ -731,3 +731,8 @@ Files extracted from upstream source:
 
 - lib/{common/,compress/,decompress/,zstd.h}
 - LICENSE
+
+## SMOLV
```

Comment:
Upstream calls it SMOL-V (with a hyphen):

```suggestion
## SMOL-V
```

Reply:
Actually this file lists dependencies based on their paths, so this should go in the `## misc` section.